### ISSUE TITLE
Looking in wrong folder for assets

### ISSUE THREAD

**Issue by ScottPierce** [robolectric/robolectric#2900]
Robolectric seems to be looking in the wrong folder for the asset of this library: https://github.com/JakeWharton/ThreeTenABP

```
Caused by: java.io.FileNotFoundException: build/intermediates/bundles/internalMultiDex/debug/assets/org/threeten/bp/TZDB.dat (No such file or directory)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at org.robolectric.res.FileFsFile.getInputStream(FileFsFile.java:84)
	at org.robolectric.shadows.ShadowAssetManager.open(ShadowAssetManager.java:324)
	at android.content.res.AssetManager.open(AssetManager.java)
	at com.jakewharton.threetenabp.AndroidThreeTen.init(AndroidThreeTen.java:27)
	... 23 more
```

It should be looking in
`build/intermediates/assets/internalMultiDex/debug`

**Comment by davidtcdeveloper** [comment]
I am facing a similar situation here:
`java.io.FileNotFoundException: build/intermediates/bundles/development/automation/assets/sql/v2000.sql (No such file or directory)`

I can find the assets under `build/intermediates/assets/development/automation`.

Using Android Gradle Plugin `2.3.3`, build tools `25.0.3` and Robolectric `3.5.1`.

I can provide more data if needed.

**Comment by xian** [comment]
This should be fixed with [Robolectric 4.0](http://robolectric.org/migrating/#migrating-to-40) in binary resources mode; please reopen if not. Thanks!
